sfcsample/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8)
+
+SET(CMAKE_BUILD_TYPE "Debug")
```

Comment:
why set as Debug ?

Reply:
Delete CMakeLists.txt, because utils use automake rather than cmake, and we already add Makefile.am to the sample code.